pkg/plugins/golang/v3/scaffolds/webhook.go

```diff
@@ -37,26 +37,22 @@ type webhookScaffolder struct {
 	boilerplate string
 	resource    *resource.Resource
 
-	// Webhook type options.
-	defaulting, validation, conversion bool
+	// Webhooks type options.
+	webhook config.Webhooks
```

Comment:
Why not plural? This is actually the struct that determines which webhooks are implemented.

Reply:
Good nit catcher.